packages/fast-components-react-base/src/button/button.spec.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+import * as ShallowRenderer from "react-test-renderer/shallow";
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure, shallow } from "enzyme";
+import examples from "./examples.data";
+import { generateSnapshots } from "@microsoft/fast-jest-snapshots-react";
+import {
+    ButtonHTMLTags,
+    ButtonProps,
+    IButtonClassNameContract,
+    IButtonHandledProps,
+    IButtonManagedClasses,
+    IButtonUnhandledProps
+} from "./button";
+
+/*
+ * Configure Enzyme
+ */
+configure({adapter: new Adapter()});
+
+describe("button", (): void => {

```

Comment:
it might be nice if this was "button snapshot" and below was "button" because of the way it gets read in the console when running, it will then look like:

button:
 - should return an object ... etc

Reply:
Great suggestion, I like this convention.